DynamicGapPlanner:
    # Robot
    robot_radius: 0.25 # m
    max_vel_x: 0.40
    max_vel_y: 0.40
    max_vel_theta: 0.8
    max_range: 5.0 # m

    # PlanningMode
    holonomic: false
    projection_operator: true
    egocircle_prop_cheat: false
    gap_feasibility_check: true
    perfect_gap_models: false
    future_scan_propagation: true

    # Control
    man_ctrl: false
    ctrl_ahead_pose: 2

    # Goal
    xy_goal_tolerance: 0.20
    yaw_goal_tolerance: 6.28
    xy_waypoint_tolerance: 0.1

    # Gap Association
    assoc_thresh: 1.0 # 0.50

    # Gap Manipulation
    rgc_angle: 1.0

    # Trajectory
    integrate_maxt: 5.0 # 7.5 # 
    integrate_stept: 0.5
    max_pose_to_scan_dist: 0.5
    Q: 1.0
    pen_exp_weight: 5.0
    inf_ratio: 1.25
    Q_f: 1.0

    # ProjectionParam
    k_po_x: 1.0
    r_unity: 0.35
    r_zero: 1.0